fuzzing/fuzz_TestSpecParser.cpp

```diff
@@ -0,0 +1,16 @@
+//License: Boost 1.0
+//By Paul Dreik 2020
+
+#include "catch2/internal/catch_test_spec_parser.hpp"
+#include "catch2/internal/catch_tag_alias_registry.hpp"
```

Comment:
```suggestion
#include "catch2/internal/catch_tag_alias_registry.hpp"
```
```suggestion
#include <catch2/internal/catch_test_spec_parser.hpp>
#include <catch2/internal/catch_tag_alias_registry.hpp>
```

Just a style nitpick